code/game/objects/items/handcuffs.dm

```diff
@@ -29,20 +29,6 @@
 	if(!C.handcuffed)
 		place_handcuffs(C, user)
 
-/obj/item/handcuffs/obj/structure/MouseDrop(mob/living/carbon/human/H)
```

Comment:
cursed

Reply:
It is, that's why i'm getting rid of it on the spot